src/code/templates/fv-egg-game.js

```diff
@@ -1287,7 +1287,7 @@ export default class FVEggGame extends Component {
   }
 
   componentWillUnmount() {
-    this.props.onResetGametePools();
+    this.props.onResetGametes();
```

Comment:
This was changed in commit d8a8f6473de220d85df6357d3e6df1b412e0be14 from onResetGametes() to onResetGametePools() without explanation. I'm proposing to change it back.